### DISCUSSION TITLE
Does capacitor storage api only use webview local storage?

### DISCUSSION THREAD

**Discussion by wudobi** [ionic-team/capacitor#4671] [Q&A]
i'm using capacitor2
i wanna save native storage(ios, android) not webview local storage.

**Comment by jcesarmobile** [comment]
No, it uses [NSUserDefaults](https://developer.apple.com/documentation/foundation/nsuserdefaults) on iOS and [SharedPreferences](https://developer.android.com/reference/android/content/SharedPreferences?hl=es) on android
